contracts/strategies/mstable/IdleMStableStrategy.sol

```diff
@@ -123,8 +131,10 @@ contract IdleMStableStrategy is Initializable, OwnableUpgradeable, ERC20Upgradea
     /// @param _extraData must contain the minimum liquidity to receive, start round and end round round for which the reward is being claimed
     /// @return rewards amount of reward that is deposited to vault
     function redeemRewards(bytes calldata _extraData) external override onlyIdleCDO returns (uint256[] memory rewards) {
-        (uint256 minLiquidityTokenToReceive, uint256 startRound, uint256 endRound) = abi.decode(_extraData, (uint256, uint256, uint256));
-        _claimGovernanceTokens(startRound, endRound);
+        (uint256 minLiquidityTokenToReceive, uint256 endRound) = abi.decode(_extraData, (uint256, uint256));
```

Comment:
I think it would be better to avoid passing `endRound` all together and always fetch it via the `unclaimedRewards` call, or do you think that it can take too much gas if not called for a while? Alternatively I think you could add an `if (endRound) == 0)` then fetch the value using `unclaimedRewards` so we have both possibilities available with the 'default' one being the call to unclaimedRewards